ecs_helper/cmd/pact_tags/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"os"
+	"fmt"
+	"net/http"
+	"encoding/base64"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+)
+
+func basicAuth(username, password string) string {
+	auth := username + ":" + password
+	return base64.StdEncoding.EncodeToString([]byte(auth))
+}
+
+func getPactPassword(roletoassume string, pactsecretkey string) string {
+	mysession := session.Must(session.NewSession())
```

Comment:
Generally in Go code, multi-word var names are in camel case to aid readability (https://golang.org/doc/effective_go.html#mixed-caps). I'm not sure if we can enforce this with `gofmt` somehow but worth looking into.

Reply:
gofmt doesn't do it but doing it manually :)